src/anonymousUserTracking/criteriaCompletionChecker.ts

```diff
@@ -374,14 +374,14 @@ class CriteriaCompletionChecker {
           return this.evaluateComparison(
             query.comparatorType,
             valueFromObj,
-            query.value ? query.value : ''
+            query.value ?? query.values ?? ''
```

Comment:
what happens if an empty string is evaluated? should we even run the evaluation at all if `value` or `values` isn't present?

Reply:
So empty string is compared with the criteria value, empty string is not matched with value and gives result as false. 
 Yes, we run the evaluation.